src/components/ApplyTabs.tsx

```diff
@@ -0,0 +1,189 @@
+import React from 'react';
+import { makeStyles, withStyles, createStyles } from '@material-ui/core/styles';
+import { Box, Tabs, Tab, Typography, Button } from '@material-ui/core';
+
+import CodeIcon from '@material-ui/icons/Code';
+import BrushIcon from '@material-ui/icons/Brush';
+
+const StyledTabs = withStyles({
+  root: {
+    alignItems: 'left',
+  },
+  indicator: {
+    backgroundColor: '#828282',
+    right: 165,
+    width: 10,
+  },
+})(Tabs);
+
+const StyledTab = withStyles((theme) =>
+  createStyles({
+    root: {
+      textTransform: 'none',
+      fontWeight: theme.typography.fontWeightRegular,
+      fontSize: theme.typography.pxToRem(12),
+
+      marginRight: theme.spacing(1),
+      '&:focus': {
+        opacity: 1,
+      },
+      padding: 0,
+    },
+    wrapper: {
+      // display: 'inline',
+      // paddingLeft: '1rem',
+      // paddingRight: '1rem'
+    },
+  })
+)(Tab);
+
+// From MUI docs on vertical tabs
+
+interface TabPanelProps {
+  children?: React.ReactNode;
+  index: number;
+  value: number;
+}
+
+function TabPanel(props: TabPanelProps) {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`vertical-tabpanel-${index}`}
+      aria-labelledby={`vertical-tab-${index}`}
+      {...other}
+    >
+      {value === index && (
+        <Box pl={2}>
+          <Typography>{children}</Typography>
+        </Box>
+      )}
+    </div>
+  );
+}
+
+function a11yProps(index: number) {
+  return {
+    id: `vertical-tab-${index}`,
+    'aria-controls': `vertical-tabpanel-${index}`,
+  };
+}
+
+const useStyles = makeStyles(() => ({
+  root: {
+    flexGrow: 1,
+    display: 'flex',
+    height: '100%',
+    minHeight: '80vh',
+    paddingTop: 0,
+  },
+  tabs: {
+    minWidth: 'fit-content',
+  },
+  tab: {},
+  list: {
+    marginTop: 5,
+  },
+}));
+
+const VerticalTabs: React.FC = () => {
+  const classes = useStyles();
+  const [value, setValue] = React.useState(0);
+
+  const handleChange = (
```

Comment:
Could you add a comment explaining why we need this? 

Also, why is event listed if it's unused?

Reply:
Honestly, ill just delete the ApplyTabs component. We arent going to use it anyway